### DISCUSSION TITLE
DiscordSRV Integration

### DISCUSSION THREAD

**Discussion by dytplay** [Zrips/CMI#3032] [Q&A]
I use the DiscordSRV plugin to make it show which chat the player wrote to, global or local. So all messages go to the same dissonance channel, but have an external difference

There is a setting to make near chat for a certain number of blocks. And a message to the entire server ( when using ! at the beginning of the message). Here's how to divide them in the channel of discord?

I contacted the developers of DiscordSRV, they said that you are engaged in integration.
Thank you in advance

**Comment by Zrips** [comment]
Well question would be quite simple: Who needs to see message coming from discord? 
Because in normal situation we can show it to everyone, but if you have range enabled, then who should see it? And from where? Everyone or some small group around some point?

**Comment by dytplay** [comment]
I want to make a separate channel in the discord for moderators to see what the players are talking about. Want to do to see what a player wrote in local chat and in global. Something like "socialspy".

Now the messages that come from minecraft to discord are not visually different. Is it possible to add any differences? ( For example, the same prefix " Socialspy" )

**Comment by dytplay** [comment]
Help pls

**Comment by CloudeLecaw** [comment]
not possible?
CMI just offers you one channel for Discord interegration ;)
So for more you need to add a feature request like we all do with wishes^^
